package.json

```diff
@@ -15,7 +15,9 @@
         "i18n:upgrade": "proton-i18n extract && proton-i18n crowdin -u && proton-i18n commit update",
         "pretty": "prettier --write  $(find src/app -type f -name '*.js')",
         "preversion": "git update-index --no-assume-unchanged package-lock.json",
-        "postversion": "git update-index --assume-unchanged package-lock.json && git push --tags"
+        "postversion": "git update-index --assume-unchanged package-lock.json && git push --tags",
+        "e2e": "cypress verify && cypress run",
+        "e2e-dev": "cypress verify && cypress open"
```

Comment:
why verify, it's done post install and it's slow.

Reply:
I was trying to understand what cypress verify does and found few people complaining about post install cypress verify failures. At the time, adding this seemed like a good idea.

Reply:
If you think of the speed as an issue, I think we can remove it from e2e(CLI) command.

Reply:
It's done post install so yep we can remove it. At least during the dev it's important to don't have it.